x-pack/plugins/infra/server/lib/metrics/lib/calculate_bucket_size/unit_to_seconds.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { sortBy, isNumber } from 'lodash';
+import { INTERVAL_STRING_RE } from './interval_regex';
+
+export const ASCENDING_UNIT_ORDER = ['ms', 's', 'm', 'h', 'd', 'w', 'M', 'y'];
+
+const units: Record<string, number> = {
```

Comment:
Would using `unitsMap` from `elastic-datemath` work better here?

Reply:
It doesn't seem to have anything defined beyond `d`

Reply:
I think we should just stick with this, it's been around for a long time and it's tried and true.